src/python/py/models/builder.py

```diff
@@ -2859,6 +2920,7 @@ def get_args():
                     is the prerequisite for the CUDA graph to be used correctly. It is not guaranteed that cuda graph be enabled as it depends on the model
                     and the graph structure.
                 use_8bits_moe = 1 : Use 8-bit quantization for MoE layers. Default is using 4-bit quantization.
+                use_qdq = 1 : Use the QDQ decomposition for quantized MatMul instead of the MatMulNBits operator.
```

Comment:
Can you add a section that shows example usage in the README after the MoE section [here](https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/py/models/README.md#use-8-bits-quantization-in-qmoe)?